metrics/ci_integrations/lib/ci_integration/ci_integration.dart

```diff
@@ -43,7 +43,12 @@ class CiIntegration {
       }
 
       if (newBuilds != null && newBuilds.isNotEmpty) {
-        await storageClient.addBuilds(storageProjectId, newBuilds);
+        await storageClient.addBuilds(
+          storageProjectId,
+          newBuilds
+              .map((build) => build.copyWith(projectId: storageProjectId))
+              .toList(),
```

Comment:
Maybe move this to a variable and pass it to the `addBuilds` method? I suppose this should improve readability - what do you think?